production.ini.in

```diff
@@ -21,7 +21,8 @@ pyramid.includes = pyramid_tm
 sqlalchemy.are.url = postgresql://${dbhost}:${dbport}/are_${dbstaging}
 sqlalchemy.bafu.url = postgresql://${dbhost}:${dbport}/bafu_${dbstaging}
 sqlalchemy.bak.url = postgresql://${dbhost}:${dbport}/bak_${dbstaging}
-sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_${dbstaging}
+#sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_${dbstaging}
+sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_ltpin_master
```

Comment:
please revert

Reply:
It's on my todo list before the deploy. I am still waiting for the final feedback of the Datenherren and after that I will change everything. Everything is: 
- revert changes in `production.ini.in`
- adapt paths from `json/test/*` to `json/*` in mapfile and json-stylesheet once the definitive data is moved to the correct folder.